<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupInfo</class>
 <widget class="QDialog" name="GroupInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>群资料</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>239</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="EditButton">
      <property name="text">
       <string>修改资料</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SaveButton">
      <property name="text">
       <string>保存修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CloseButton">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>321</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>群介绍</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="IntroEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>321</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>群公告</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="AnnounceEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>群名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="NameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>创建时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>68</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>群等级</string>
   </property>
  </widget>
  <widget class="QLabel" name="GroupNum">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>31</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1016862375</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>31</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>群号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>68</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Images/Level.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="SetupTime">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>105</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2015-6-30</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GroupFaceBtn">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>142</y>
     <width>72</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Images/12.png</normaloff>:/Images/12.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
